test/unit/helpers/dom/Dom.js

```diff
@@ -21,9 +21,16 @@ const Dom = Class.extend({
 });
 
 Dom.isVisible = function($el) {
-  // non-jquery method
-  // return $el.css('visibility') === 'visible' && $el.css('display') !== 'none' && $el.html() !== '';
-
+  // jsdom has issue with :visible selector
+  // check visibility recursively instead
+  if (global.useJest) {
```

Comment:
can we remove this conditional and the "jquery method" now? i see the jest setup file sets this true